conf.d/go-metro.yaml.example

```diff
@@ -13,9 +13,9 @@
 #        Also, please note that go-metro logs to its own file - found in /var/log/datadog/go-metro.log.
 #        Additionally, go-mtro runs standalone so it will *NOT* currently appear on the agent's info page.
 #
-#        Finally, because the go-metro binary is only bundled with the RPM and DEB distributions of the
+#        Finally, because the go-metro binary is only bundled with the 64-bit RPM and DEB distributions of the
 #        datadog-agent, it will only be available in those packaged versions (ie. go-metro is currently
-#        not available with the source install).
+#        available neither with the source install nor with the 32-bit packages).
```

Comment:
maybe better with `currently unavailable with the source install or the 32-bit packages`...